tests/test_app_client.py

```diff
@@ -630,6 +637,16 @@ async def test_delete_fragment(self, service: MockApp):
             await client.delete_fragment(fragment_id=ID)
             assert service.id == ID
 
+    @pytest.mark.asyncio
+    async def test_get_fragment_history(self, service: MockApp):
+        async with ChannelFor([service]) as channel:
+            client = AppClient(channel, METADATA, ID)
+            fragment_history = await client.get_fragment_history(id=ID, page_token="", page_limit=20)
+            assert service.fragment.id == ID
```

Comment:
Can we also add `assert service.id == ID` and similar for `page_token` and `page_limit`?